public/app/features/dashboard/components/Inspector/QueryInspector.tsx

```diff
@@ -202,6 +251,7 @@ export class QueryInspector extends PureComponent<Props, State> {
             new query. Hit refresh button below to trigger a new query.
           </p>
         </div>
+        {executedQueries.length && this.renderExecutedQueries(executedQueries)}
```

Comment:
nit: maybe move the length check to `this.renderExecutedQueries`?
```suggestion
        {this.renderExecutedQueries(executedQueries)}
```